netlify.toml

```diff
@@ -33,3 +34,4 @@
     Referrer-Policy = "strict-origin-when-cross-origin"
     Strict-Transport-Security = "max-age=2592000"
     Permissions-Policy = "vibrate=(), geolocation=(), midi=(), notifications=(), push=(), microphone=(), camera=(), magnetometer=(), gyroscope=(), speaker=()"
+    X-Robots-Tag = "noindex, nofollow"
```

Comment:
Added this here and will adjust when we're ready to deal with robots/sitemaps etc